src/wp-includes/media.php

```diff
@@ -1137,6 +1137,15 @@ function wp_get_attachment_image( $attachment_id, $size = 'thumbnail', $icon = f
 			}
 		}
 
+		// Adds auto to the sizes attribute if applicable.
+		if (
+			( isset( $attr['loading'] ) && 'lazy' === $attr['loading'] ) &&
+			isset( $attr['sizes'] ) &&
+			! wp_sizes_attribute_includes_valid_auto( $attr['sizes'] )
```

Comment:
Unnecessary parentheses here and a bit confusing to not have every clause in its own line.
```suggestion
			isset( $attr['loading'] ) &&
			'lazy' === $attr['loading'] &&
			isset( $attr['sizes'] ) &&
			! wp_sizes_attribute_includes_valid_auto( $attr['sizes'] )
```